packages/fxa-payments-server/src/store/utils.tsx

```diff
@@ -39,12 +57,18 @@ export const apiFetch = (
       'Authorization': `Bearer ${accessToken}`,
       ...options.headers || {}
     },
-  }).then(response => {
-    if (response.status >= 400) {
-      throw new APIError(response, 'status ' + response.status);
-    }
-    return response.json();
   });
+  if (response.status >= 400) {
+    let body = {};
+    try {
+      // Parse the body as JSON, but will fail if things have really gone wrong
+      body = await response.json();
```

Comment:
More detailed errors bubbling up from subhub, attempting to make them usable here.